components/results_table/commons/results_table.lua

```diff
@@ -5,3 +5,109 @@
 --
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+---Note: This can be overwritten
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+---Note: This can be overwritten
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+
+local DEFAULT_VALUES = {
+	order = 'desc',
+	playerPrefix = 'p',
+	coachPrefix = 'c',
+	resolveOpponent = true,
+	playerLimit = 10,
+	coachLimit = 5,
+	achievementsLimit = 10,
+	resultsLimit = 5000,
+}
+local COACH_TYPE = 'coach'
+local TYPE_ALIASES = {
```

Comment:
This is MODE_ALIASES right?

Reply:
query types
but i am considering ditching the aliases altogether to simplify